src/app/garfile/craft/validations.js

```diff
@@ -13,15 +17,15 @@ module.exports.validations = (craftObj) => {
 
   return [
     [
-      new ValidationRule(validator.notEmpty, 'craftReg', registration, 'Enter a registration'),
+      new ValidationRule(validator.notEmpty, 'craftReg', registration, i18n.__('validation_aircraft_registration')),
```

Comment:
Externalised validation messages for aircraft.

This will need to be applied to the rest of the app, which will require a bit of time rather than special effort.